ruby/ext/google/protobuf_c/defs.c

```diff
@@ -307,19 +501,21 @@ void Descriptor_register(VALUE module) {
 
 /*
  * call-seq:
- *    Descriptor.new(file_descriptor)
+ *    Descriptor.new(c_only_cookie, ptr) => Descriptor
  *
- * Initializes a new descriptor and assigns a file descriptor to it.
+ * Creates a descriptor wrapper object.  May only be called from C.
```

Comment:
Should?

Reply:
I think "may" is better. Calling from Ruby is prohibited, not just recommended against.